### DISCUSSION TITLE
[BUG]: updateListing deletes translations

### DISCUSSION THREAD

**Discussion by devtronic** [etsy/open-api#712] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

We need to call `PATCH updateListing` to mark a listing as `inactive` when the stock amount has reached 0.
After we patched the listing with `state=inactive` all translations except the english translation are gone.

PATCH requests should "**_Partially_** update resource’s state." (https://en.wikipedia.org/wiki/Representational_state_transfer#Applied_to_web_services)

### Steps to reproduce

1. Create an etsy account, the default language **must not EN**. (In this specific case it was a seller with a german account)
2. Create a product with a title and description in the default language (=DE)
3. The product should also have a title + description which was translated automatically to EN.
4. Call  `PATCH shops/{shop_id}/listings/{listing_id}` with the body `state=inactive`
5. The title and description of the product is now EN

### Expected behavior

The translations are not touched during the state change because PATCH should update a resource **_partially_** and the body doesn't contain `title` or `description`.

**Comment by jonbelaire** [comment]
We have also been experiencing this issue!

**Comment by devtronic** [comment]
Update: When the seller deactivate the automatic translations and reactivate it, the title and description stays in the original language (DE in our case).

I don't know how this workaround affects new listings which are created after reactivating the automatic translations.

**Comment by etsyachristensen** [comment]
@devtronic and @jeriale This has been logged in our system for review.

**Comment by etsyachristensen** [comment]
We are closing the issues board in favor of the discussion board. We'll convert this to a new discussion there to continue tracking it.
